Vialer/VoIPGRIDRequestOperationManager.m

```diff
@@ -338,6 +340,33 @@ - (NSString *)sipPassword {
     return nil;
 }
 
+- (void)pushMobileNumber:(NSString *)mobileNumber success:(void (^)())success  failure:(void (^)(NSError *error, NSString *userFriendlyErrorString))failure {
+    //Strip whitespaces
+    mobileNumber = [mobileNumber stringByReplacingOccurrencesOfString:@" " withString:@""];
+    //replace leading zero with +31
+    if ([mobileNumber hasPrefix:@"0"])
+        mobileNumber = [NSString stringWithFormat:@"+31%@", [mobileNumber substringFromIndex:1]];
+    
+    NSDictionary *parameters = @{@"mobile_nr" : mobileNumber};
+    
+    if ([mobileNumber length] > 0) {
```

Comment:
This check should be above parameters dictionary so it won't crash at nil.
For efficiency you could set the if around the whole block.